app/src/modules/settings/routes/data-model/field-detail/store/alterations/global.ts

```diff
@@ -77,3 +77,10 @@ export function resetRelations(updates: StateUpdates) {
 	updates.relations.m2o = undefined;
 	updates.relations.o2m = undefined;
 }
+
+export function resetInterfaceAndDisplay(updates: StateUpdates) {
+	set(updates, 'field.meta.interface', undefined);
+	set(updates, 'field.meta.options', undefined);
+	set(updates, 'field.meta.display', undefined);
+	set(updates, 'field.meta.display_options', undefined);
+}
```

Comment:
So I noticed this somewhat weird behavior when attempting to select `directus_files` in an M2O field.

https://github.com/user-attachments/assets/560e1aff-050b-4f91-a98e-ccc986a9ffa3

Now I'm aware we're trying to avoid user error here, but maybe we should adapt `resetInterfaceAndDisplay` to take in a prop so we can set the `undefined` value for `field.meta.interface` to "files"?

https://github.com/user-attachments/assets/fc3cd6b5-daa3-40b0-9207-8fb5516f1546

Reply:
@Nitwel, see above, forgot to tag

Reply:
Made it now that it switches to the File/Files interface instead.